insights/api/insight_api.py

```diff
@@ -0,0 +1,80 @@
+import os
+import asyncio
+from datetime import datetime
+
+import bittensor as bt
+from insights.api.query import TextQueryAPI
+from insights.api.get_query_axons import get_query_api_axons
+from neurons.validators.utils.uids import get_top_miner_uid
+from fastapi import FastAPI, Body
+import uvicorn
+
+
+bt.debug()
+
+def main():
+    app = FastAPI()    
+    wallet_name = os.getenv("WALLET_NAME")
+    wallet_hotkey = os.getenv("WALLET_HOTKEY")
+    netuid = os.getenv("NETUID", str(15))
+    
+    wallet = bt.wallet(name=wallet_name, hotkey=wallet_hotkey)
+    
+    text_query_api = TextQueryAPI(wallet=wallet)
+    
+    @app.get("/api/text_query")
+    async def get_response(network:str, text: str):
+        # select top miner
+        metagraph = bt.subtensor("local").metagraph(netuid=netuid)
```

Comment:
dont hardcode local, make it configurable

Reply:
make subtensor, api port, wallet and netuid configurable